Test suite fixes

sjchmiela opened the pull request:
# Why

Test suite is failing hard ATM.

# How

Debug each issue and fix.

# Test Plan

`test-suite` works better now.

sjchmiela committed:
[tests] Fix a couple of issues in Expo SDK and test-suite

- Add .db as an asset extension of expo-yarn-workspaces
- Fix NSNumber issues in EXFileSystem when manipulating files with Base64 encoding
- Fix timing issues in test-suite

ide commented on apps/test-suite/tests/FileSystem.js:
```diff
@@ -60,25 +59,25 @@ export function test(t) {
         const b64 = await FS.readAsStringAsync(asset.localUri, options);
         t.expect(b64).toBeDefined();
         t.expect(typeof b64).toBe('string');
-        t.expect(b64.split('') % 4).toBe(0);
```
lol

ide commented on packages/expo-yarn-workspaces/index.js:
```diff
@@ -44,6 +45,8 @@ exports.createReactNativeConfiguration = function createReactNativeConfiguration
     watchFolders,
 
     resolver: {
+      assetExts: assetExts.concat(['db']),
```
`[...assetExts, 'db']` (we are guaranteed Node 8+ here)

ide reviewed (approved):
Thanks so much for fixing these things

sjchmiela committed:
Fix review comment

sjchmiela committed:
Run yarn install v1.10.1
[1/4] Resolving packages...
success Already up-to-date.
$ expo-module prepare
Done in 3.25s.
